internal/pkg/aws/ec2/ec2.go

```diff
@@ -57,27 +57,38 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
-// GetSubnetIDs finds the subnet IDs with optional filters.
-func (c *EC2) GetSubnetIDs(filters ...Filter) ([]string, error) {
-	inputFilters := toEC2Filter(filters)
+// SubnetIDs finds the subnet IDs with optional filters.
+func (c *EC2) SubnetIDs(filters ...Filter) ([]string, error) {
+	subnets, err := c.subnets(filters...)
+	if err != nil {
+		return nil, err
+	}
 
-	response, err := c.client.DescribeSubnets(&ec2.DescribeSubnetsInput{
-		Filters: inputFilters,
-	})
+	subnetIDs := make([]string, len(subnets))
+	for idx, subnet := range subnets {
+		subnetIDs[idx] = aws.StringValue(subnet.SubnetId)
+	}
+	return subnetIDs, nil
+}
 
+// PublicSubnetIDs finds the public subnet IDs with optional filters.
```

Comment:
nit: This applies to all of the method comments. Since this is already above the method, no reason to restate the method name

`Returns the public subnet IDs with optional filters.`